src/features/timeline/TimelineIndexPage.js

```diff
@@ -40,28 +43,63 @@ const CycleIndexPage = ({ entryId }) => {
     })
   }, [selectedDate])
 
+  const daysInMonths = range.reduce((acc, curr) => {
+    const month = getMonth(curr)
+    if (acc[month]) {
+      acc[month] = [...acc[month], curr]
+    } else {
+      acc[month] = [curr]
+    }
+    return acc
+  }, {})
+
   return (
     <AppLayout>
       <AppPage>
         <AppMainToolbar>
           <DatePicker date={selectedDate} />
           <IconButton
             aria-label="Scroll to today"
-            onClick={() => navigate(`/timeline/${entryIdFromDate(new Date())}`)}
+            onClick={() =>
```

Comment:
**nitpick:** This seems to not work, goes back to timeline view when clicked if on calendar

Reply:
Fixed :D